build/localmode/RunSamples.cmd

```diff
@@ -13,6 +19,13 @@ if "%1" == "" (
         set VERBOSE="verbose"
         @echo [RunSamples.cmd] VERBOSE is !VERBOSE!
     )
+
```

Comment:
Another option. Have RunSparkCLR.cmd that takes exe as the parameter. RunSamples can just call RunSparkCLR with SparkCLRSamples.exe as arg. For other scenario is including examples, users may use RunSparkCLR directly.